src/Root.cpp

```diff
@@ -53,7 +53,6 @@ cRoot * cRoot::s_Root = nullptr;
 
 
 cRoot::cRoot(void) :
-	m_pDefaultWorld(nullptr),
```

Comment:
Should probably still be initialising here?

Reply:
There's always a default world, so it's initialised by LoadWorlds

Reply:
Which isn't run in the constructor, so there is period where it is uninitialised.

Reply:
Sure, but LoadWorlds happens before any plugin, object attempts to call GetDefaultWorld. Even if it didn't, every caller assumes the return value can be used immediately/stored, so it's undefined behaviour anyway if that's violated.

Reply:
With a `nullptr` you can easily see in the debugger that it's uninitialized. If it's just a random garbage pointer then you don't know if it's uninitialized or if there's memory corruption.

Reply:
Wouldn't you agree though this will never happen, by the above reasoning? So this line of code is unneeded.

Reply:
It is a good practice to never leave any pointer uninitialized.

Reply:
I suppose I can add a debug check for it.